src/Compilers/Test/Utilities/CSharp/BasicCompilationUtils.cs

```diff
@@ -37,7 +37,9 @@ private static VisualBasicCompilation CreateCompilationWithMscorlib(string sourc
             return VisualBasicCompilation.Create(assemblyName, new[] { tree }, references, options);
         }
 
-        private static BasicTestBase s_instance = new BasicTestBase();
+        private static BasicTestBase s_instance;
+
+        private static BasicTestBase Instance => s_instance ?? (s_instance = new BasicTestBase());
```

Comment:
@AlekseyTs if we depend on object identity that itself would be a bug here because it would make running tests in parallel a problem.